service/api_logic/games_logic.py

```diff
@@ -21,7 +21,9 @@
 def filter_matches_by_date(matches, today, date_key=DATE_KEY):
```

Comment:
I see the default value being set to 'DATE_KEY', but I can hardly see any potential usage for that